.github/workflows/cdn.yml

```
@@ -0,0 +1,38 @@
+name: Publish to CDN
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "The release tag to use"
+        required: true
+      bucket:
+        description: "S3 bucket to upload to"
+        default: "prod-cdn.ably.com"
+        type: choice
+        options:
+          - "prod-cdn.ably.com"
+          - "nonprod-cdn.ably.com"
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    # These permissions are necessary to run the configure-aws-credentials action
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.version }}
+      - uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: arn:aws:iam::${{ secrets.ABLY_AWS_ACCOUNT_ID_PRODUCTION }}:role/github-actions-sdk
```

The `prod-cdn.ably.com` S3 bucket is in the SDK account, so [similar to ably-js](https://github.com/ably/ably-js/blob/main/.github/workflows/publish-cdn.yml#L44), this needs to be:

```suggestion
          role-to-assume: arn:aws:iam::${{ secrets.ABLY_AWS_ACCOUNT_ID_SDK }}:role/prod-ably-sdk-cdn
```

(I just added the `ABLY_AWS_ACCOUNT_ID_SDK` secret)

---

Fixed in d86a4811f1032cf39887a42d377635229e21ca76